static/js/settings_users.js

```diff
@@ -28,7 +32,10 @@ function update_view_on_deactivate(row) {
 
 function update_view_on_reactivate(row) {
     var button = row.find("button.reactivate");
+    var user_role = row.find(".user_role");
     row.find("button.open-user-form").show();
+    row.find('i.deactivated-user-icon').hide();
+    user_role.text(user_role.text().replace(/Deactivated|\(|\)/g, ""));
```

Comment:
This parsing logic is messy, since the correctness depends on us always alternating calls to these two functions.

It's also broken with i18n where "Deactivated" will be a different word here; I think it might be better to either recompute or have a `data-orig-role` value that we can just pull out here.  

We could compute more cleanly above, too).